src/App.js

```diff
@@ -29,30 +29,41 @@ class App extends Component {
       this.getActivePlayers(allPlayers)
     }
   }
-  
+
+  showFrontOrBack = (sideToShow) => {
+      this.setState({ showCardSide: sideToShow})
+  }
```

Comment:
This is the function that sets which side of the card should be showing. Might require a refactor ... might turn it back into a boolean now that there's only two options.